docker-compose-addons.yml

```diff
@@ -100,11 +100,11 @@ services:
       PG_DB_PASS: ${PG_DB_PASS}
 
       PROJECT_ID: ${GCP_PROJECT_ID}
-      GOOGLE_APPLICATION_CREDENTIALS: ${GOOGLE_APPLICATION_CREDENTIALS}
+      GOOGLE_APPLICATION_CREDENTIALS: /home/gcp_key.json
 
       LOGGING_LEVEL: ${ISS_LOGGING_LEVEL}
     volumes:
-      - ${GOOGLE_APPLICATION_CREDENTIALS}:/google/gcp_credentials.json
+      - ./resources/google/${GOOGLE_ACCESS_KEY_NAME}:/home/gcp_key.json
```

Comment:
Question: Can we remove the GOOGLE_APPLICATION_CREDENTIALS env var from the sample.env file(s) related to the firmware manager now that we no longer reference it here?

Reply:
I have removed it from the sample.env file on the root level. I have kept this variable in the sample.env within the services/api/ directory since if you are running the application as standalone it will need to be set.